src/nft/utils/asset.ts

```diff
@@ -44,3 +45,36 @@ export const generateTweetForPurchase = (assets: UpdatedGenieAsset[], txHashUrl:
   } with @Uniswap 🦄\n\nhttps://app.uniswap.org/#/nfts/collection/0x60bb1e2aa1c9acafb4d34f71585d7e959f387769\n${txHashUrl}`
   return `https://twitter.com/intent/tweet?text=${encodeURIComponent(tweetText)}`
 }
+
+function getMinListingPrice(listings: Listing[]): number {
+  return min(listings.map((listing) => listing.price ?? 0)) ?? 0
```

Comment:
you don't need to import from `d3` you can just do `return Math.min(...listings.map((listing) => listing.price ?? 0)) ?? 0`